x-pack/plugins/infra/public/locators/logs_locator.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { LocatorDefinition, LocatorPublic } from '@kbn/share-plugin/public';
+import type { SerializableRecord } from '@kbn/utility-types';
+import type { InfraClientCoreSetup, QueryTimeRange } from '../types';
+import { DISCOVER_APP_TARGET } from '../../common/constants';
+
+const LOGS_LOCATOR_ID = 'LOGS_LOCATOR';
+
+export interface LogsLocatorParams extends SerializableRecord {
+  /** Defines log position */
+  time?: number;
+  /**
+   * Optionally set the time range in the time picker.
+   */
+  timeRange?: QueryTimeRange;
+  filter?: string;
+  logViewId?: string;
```

Comment:
How about we define this as [`logView: LogViewReference`](https://github.com/mohamedhamed-ahmed/kibana/blob/d5e3554e52b843625f95e2fa8a1c105b1680374f/x-pack/plugins/infra/common/log_views/types.ts#L122) so we can support log views by value and by reference?